DQM/L1TMonitor/src/L1TdeRCT.cc

```diff
@@ -122,20 +123,6 @@ L1TdeRCT::L1TdeRCT(const ParameterSet & ps) :
   if (verbose_)
     std::cout << "L1TdeRCT: constructor...." << std::endl;
 
-  outputFile_ =
-      ps.getUntrackedParameter < std::string > ("outputFile", "");
-  if (!outputFile_.empty()) {
-    if(verbose_) std::
-  cout << "L1T Monitoring histograms will be saved to " <<
-  outputFile_.c_str() << std::endl;
-  }
-
-  bool disable =
-      ps.getUntrackedParameter < bool > ("disableROOToutput", false);
```

Comment:
If this parameter is not used anymore it should also be removed from the config files.